.github/labeler.yml

```diff
@@ -0,0 +1,16 @@
+feature:
+  - head-branch: ['^feature/']
+dependencies:
+  - head-branch: ['^dependencies/']
+enhancement:
+  - head-branch: ['^enhancement/']
+bug:
+  - head-branch: ['^bug/']
+deprecated:
+  - head-branch: ['^deprecated/']
+removed:
+  - head-branch: ['^removed/']
+security:
+  - head-branch: ['^security/']
+documentation:
+  - head-branch: ['^documentation/']
```

Comment:
Do these work with multiple? Could `docs/documentation` work for example? I personally prefer docs but then some others can't be shortened.

Same for feat/feature.

Thinking about it, perhaps its worth aligning closer with https://www.conventionalcommits.org/en/v1.0.0/ (This is what DAFNI used)?

Reply:
https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type